### PR TITLE
Add pykube-ng

### PR DESCRIPTION
**Pull request by jacobtomlinson**
Ping @conda-forge/help-python
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/pykube-ng/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,83 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (ocefpaf) [inline recipes/pykube-ng/meta.yaml:1]: Ping! Also, try to use grayskull next time. It really helps to get the recipe in good  shape for conda-forge and reduces the work of the reviewers.

File: recipes/pykube-ng/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  # More info about selectors can be found in the conda-build docs:
+  # https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#preprocessing-selectors
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build: []
```
Comment (ocefpaf) [inline recipes/pykube-ng/meta.yaml:42]:
```suggestion
```

File: recipes/pykube-ng/meta.yaml (hunk 1 above)
Comment (jacobtomlinson) [reply recipes/pykube-ng/meta.yaml]: Oh nice. Wasn't aware of that and missed the suggestion in the comments.

**Comment by ocefpaf** [comment]
@jacobtomlinson it takes ~1 hour for the CDN to sync. Then we can re-try https://github.com/conda-forge/staged-recipes/pull/18777.

**Pull request merged**